core/src/main/scala/cats/ApplicativeError.scala

```diff
@@ -273,6 +273,28 @@ trait ApplicativeError[F[_], E] extends Applicative[F] {
       case None    => raiseError(ifEmpty)
     }
 
+  /**
+   * Convert from scala.Option when the content is a value of the error type.
+   * If the option is empty, an empty unit effect is given.
+   * If the option contains an error, it is raised.
+   * or returns an empty action
```

Comment:
This `or returns an empty action` seems to be dangling, and duplicates the second sentence?